locale/pt-br/blog/announcements/welcome-redhat.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Fundação Node.js oferece boas vindas à Red Hat como a mais nova membro Platinum
+date: 2015-10-06T12:30:00.000Z
+status: publish
+category: Announcements
+slug: welcome-redhat
+layout: blog-post.hbs
+---
+
+# A Fundação Node.js oferece boas vindas à Red Hat como o mais novo membro Platinum
+
+> A empresa procura acelerar a adoção do Node.js para desenvolvimento de software corporativo
+
+**SAN FRANCISCO, 6 de outubro de 2015** - A [Fundação Node.js](https://foundation.nodejs.org/), um consórcio liderado pela comunidade e apoiado pelo mercado, para avançar o desenvolvimento da plataforma Node.js, anunciou hoje que a Red Hat, Inc. ingressou na Fundação como membro Platinum. A Red Hat se une aos membros Platinum, incluindo Famous, IBM, Intel, Joyent, Microsoft e PayPal, para fornecer apoio na adoção, desenvolvimento, e sucesso de longo prazo do projeto Node.js.
+
+O Node.js é o ambiente de execução para aplicativos de alto desempenho e baixa latência, possibilitando tudo, desde aplicativos corporativos a robôs. Nos últimos dois anos, cada vez mais, empresas grandes, incluindo Red Hat, IBM, PayPal, Fidelity e Microsoft, adotaram o Node.js como parte de sua estrutura corporativa. Hoje, existem 2 milhões de endereços IP exclusivos instalando pacotes Node.js. e mais de 2 bilhões de pacotes baixados no último mês.
+
+Freqüentemente usado para criar aplicativos de rede escaláveis e rápidos, o Node.js suporta tecnologias Red Hat, como [Red Hat Mobile Application Platform](https://www.redhat.com/en/technologies/mobile/application-platform), e está disponível em [OpenShift da Red Hat](https://www.openshift.com/) e [Red Hat Software Collections](http://developerblog.redhat.com/tag/software-collections/). Como novo membro, a Red Hat está fornecendo suporte financeiro, contribuições técnicas, e orientações de alto nível para as políticas de diretrizes da recém-criada Fundação que opera como uma organização neutra para apoiar o projeto governado pela comunidade Node.js.
```

Comment:
```suggestion
Frequentemente usado para criar aplicativos de rede escaláveis e rápidos, o Node.js suporta tecnologias Red Hat, como [Red Hat Mobile Application Platform](https://www.redhat.com/en/technologies/mobile/application-platform), e está disponível em [OpenShift da Red Hat](https://www.openshift.com/) e [Red Hat Software Collections](http://developerblog.redhat.com/tag/software-collections/). Como novo membro, a Red Hat está fornecendo suporte financeiro, contribuições técnicas, e orientações de alto nível para as políticas de diretrizes da recém-criada Fundação que opera como uma organização neutra para apoiar o projeto governado pela comunidade Node.js.
```

Reply:
i did not  catch the alteration here. could you point the line?

Reply:
@caioreis123 `Freqüentemente` replace to `Frequentemente`